.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "python.analysis.typeCheckingMode": "off",
```

Comment:
what are these for?

Reply:
i dont write the code. it's auto generate by itself